platform-sdk/swirlds-common/src/main/java/com/swirlds/common/AddressBook.java

```diff
@@ -603,7 +603,30 @@ public Set<NodeId> getNodeIdSet() {
      */
     @NonNull
     public String toConfigText() {
-        return AddressBookUtils.addressBookConfigText(this);
+        return addressBookConfigText(this);
+    }
+
+    public static String addressBookConfigText(@NonNull final AddressBook addressBook) {
```

Comment:
Is the idea that this should be replaced with a call to `AddressBookUtils` after that file has also been moved? If so, this method should probably be private, and have a FUTURE WORK note explaining the future intention